docs/site/tutorials/core/8-configuration.md

```diff
@@ -20,6 +29,9 @@ export class ChineseGreeter implements Greeter {
 
 ```
 
+In order to do that, we use the `@config` decorator to make the extension point
+to be able to receive its options.
+
```

Comment:
To be clear, we have two separate examples here:

1. ChineseGreeter that allows the style configuration
2. GreetingService that takes some options